src/neuroglancer/volume_rendering/trackable_shader_mode.ts

```diff
@@ -0,0 +1,20 @@
+import {TrackableEnum} from 'neuroglancer/util/trackable_enum';
+import {glsl_CHUNK_NUMBER_SHADER, glsl_MAX_PROJECTION_SHADER, glsl_USER_DEFINED_RAY_TRAVERSAL} from 'src/neuroglancer/volume_rendering/glsl';
+
+export enum SHADER_MODES {
+  DEFAULT = 0,
+  MAX_PROJECTION = 1,
+  CHUNK_VISUALIZATION = 2,
+}
+
+export const SHADER_FUNCTIONS = new Map([
+  [SHADER_MODES.DEFAULT, glsl_USER_DEFINED_RAY_TRAVERSAL],
+  [SHADER_MODES.MAX_PROJECTION, glsl_MAX_PROJECTION_SHADER],
+  [SHADER_MODES.CHUNK_VISUALIZATION, glsl_CHUNK_NUMBER_SHADER],
```

Comment:
There is already a "wire frame" mode option in the general "settings" panel that is intended for the same use case of debugging rendering.  I think we could just use that option to enable this mode rather than having a separate volume rendering option.

Reply:
I wasn't aware of the option, thanks. That sounds like a better solution. If it works, for this round, I'll convert the existing chunk visualisation mode to a wireframe of the chunks that are rendered instead of the ray casting.
In the future, we could maybe get that to be an overlay of the chunk wireframe on top of the ray casting instead.